src/main/java/org/folio/dcb/service/impl/EcsTlrTransactionsServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.dcb.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.domain.dto.DcbTransaction;
+import org.folio.dcb.domain.dto.TransactionStatusResponse;
+import org.folio.dcb.exception.ResourceAlreadyExistException;
+import org.folio.dcb.repository.TransactionRepository;
+import org.folio.dcb.service.EcsTlrTransactionsService;
+import org.springframework.stereotype.Service;
+import static org.folio.dcb.domain.dto.DcbTransaction.RoleEnum.BORROWER;
+import static org.folio.dcb.domain.dto.DcbTransaction.RoleEnum.LENDER;
+
+@Service
+@RequiredArgsConstructor
+@Log4j2
+public class EcsTlrTransactionsServiceImpl implements EcsTlrTransactionsService {
+
+  private final BaseLibraryService baseLibraryService;
+  private final TransactionRepository transactionRepository;
+
+  @Override
+  public TransactionStatusResponse createEcsTlrTransaction(String dcbTransactionId, DcbTransaction dcbTransaction) {
+    log.debug("createCirculationRequest:: creating new transaction request for role {} ", dcbTransaction.getRole());
```

Comment:
I think we can log this in `info`
```suggestion
    log.info("createCirculationRequest:: creating new transaction request for role {} ", dcbTransaction.getRole());
```